src/06-blackjack/Blackjack/Blackjack.Core/Deck.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Blackjack.Core
+{
+    public class Deck
+    {
+        private readonly List<Card> _cards = new List<Card>(52);
+        private string[] _suits = new string[] { "♠", "♥", "♣", "♦" };
+        private string[] _ranks = new string[] { "A","2","3","4","5","6","7","8","9","10","J","Q","K" };
+
+        public Deck()
```

Comment:
I would consider creating a constructor which will take in list of cards... Or factory methods so that the deck can be instantiated like `var deck = Deck.C36` or `var deck = D52`.. (methods can't start with a number, so prefix with 'C')

This would allow composing two most popular decks + customized deck